packages/contrast-colors/index.js

```diff
@@ -488,27 +785,46 @@ function luminance(r, g, b) {
 }
 
 function contrast(color, base, baseV) {
+  if(baseV == undefined) { // If base is an array and baseV undefined
+    let colorString = String(`rgb(${base[0]}, ${base[1]}, ${base[2]})`);
+    
+    baseLightness = Number((d3.hsluv(colorString).v));
```

Comment:
Does this come out as something other than a number? If not, you can drop `Number`.
```suggestion
    baseLightness = d3.hsluv(colorString).v;
```